micronaut-maven-plugin/src/main/java/io/micronaut/maven/testresources/TestResourcesHelper.java

```diff
@@ -345,9 +350,6 @@ private void deleteKeepAliveFile() throws MojoExecutionException {
     }
 
     private Path getServerSettingsDirectory() {
-        if (shared) {
-            return ServerUtils.getDefaultSharedSettingsPath(sharedServerNamespace);
```

Comment:
Are you sure about this removal?

Reply:
You were (as always) right! Settings were copied to the project directory, and I forgot about the ones in the home directory

Reply:
> You were (as always) right! 

My wife would disagree :)